cli/integration_tests/README.md

```diff
@@ -0,0 +1,47 @@
+## Integration Tests
+
+The `cli/integration_tests` directory contains tests for Turborepo, exercising builds of
+Turborepo against custom monorepos and turbo.json setups. Tests are written using [`prysk`][1],
+which executes the CLI and can execute arbitrary other commands to assert the result. Some tests
+assert the log output, some assert the created artifacts, some assert configuration, etc etc.
+
+### Adding new tests
+
+To add a new test to this directory, create a `my-new-test.t` file. You will likely want to
+the file to start with:
+
+```bash
+Setup
+  $ . ${TESTDIR}/setup.sh
+  $ . ${TESTDIR}/setup_monorepo.sh $(pwd)
+```
+
+- `setup.sh` sets a `TURBO` environment variable that points to the locally built binary
+- `setup_monorepo.sh` uses one of the test repos in the `_fixtures` directory to exercise
```

Comment:
How do you set a specific fixture?

Reply:
Good call. You send another argument like so:

```
$ . ${TESTDIR}/_helpers/setup_monorepo.sh $(pwd) single_package
```

I'll add in examples of this and also how to change the package manager in a new PR so I don't have to wait for tests again with another push here.